docs/adr/adr-010-send-unpublish-emails-for-single-pages.md

```diff
@@ -0,0 +1,74 @@
+# Decision record: reinstate unpublish emails for single page subscriptions
+
+Date: 2021-12-03
+
+## Context
+
+We have just launched a new email notifications feature that allows users to subscribe to updates on a single page.
+Currently this feature is only enabled on a few selected pages (eg. [Open standards for government][open-standards-for-government]),
+but it is expected to roll out to the majority of pages and content types by early 2022.
+
+Unpublished content will become unavailable and receive no further updates. Users who have demonstrated a desire to stay up to date
+with an item of content may find it unexpected if content they were subscribed to disappears without notification, or becomes unavailable through their account. This change is part of meeting a user need to stay up to date with changing content, including when that content is changed through unpublication.
+
+The email alert API used to send emails when a page in a taxon subscription was unpublished.
+This feature was [removed in Jan 2021][remove-previous-unpublish-emails] for good reasons.
+
+[open-standards-for-government]: https://www.gov.uk/government/publications/open-standards-for-government
+[remove-previous-unpublish-emails]: https://github.com/alphagov/email-alert-api/issues/1572
+
+## Decision
+
+We will reinstate emails when a page is unpublished, but only to users with a single page subscription to that page.
+We will also remove the subscriber list for a single page when it is unpublished to reduce the build up of empty lists in Email Alert API.
```

Comment:
Given the behaviour of #1694, I think a better phrasing would be

> We will add an endpoint to email-alert-api to unsubscribe everyone from a given subscriber list, and optionally send them an email with a reason why (using the existing [message](https://github.com/alphagov/email-alert-api/blob/main/docs/adr/adr-004-message-concept.md) machinery).  We will change email-alert-service to call this new endpoint when a page gets unpublished, to notify any users subscribed to that specific page.
>
> We will not delete the now-empty list, as it will be cleaned up by the [HistoricalDataDeletionWorker](https://github.com/alphagov/email-alert-api/blob/main/app/workers/historical_data_deletion_worker.rb), and introducing another mechanism that destroys lists could be a point of confusion.

As we're intentionally not adding a concept of "unpublishing" to email-alert-api.  We're adding a more general thing, which email-alert-service (which *is* aware of the publishing workflow) uses to implement the behaviour we want.